docs/rest/BulkUpdate.http

```diff
@@ -0,0 +1,497 @@
+Authorization: Bearer {{bearer.response.body.access_token}}
+Prefer: respond-async
+{
+    "resourceType": "Parameters",
+    "parameter": [
+        {
+            "name": "operation",
+            "part": [
+                {
+                    "name": "type",
+                    "valueCode": "upsert"
+                },
+                {
+                    "name": "path",
+                    "valueString": "Resource.meta"
+                },
+                {
+                    "name": "name",
+                    "valueString": "security"
+                },
+                {
+                    "name": "value",
+                    "valueCoding": {
+                        "system": "http://example.org/security-system",
+                        "code": "SECURITY_TAG_CODE",
+                        "display": "Updated Security Tag Display"
+                    }
+                }
+            ]
+        }
+    ]
+}
+
+### Bulk update operation at System level - Only the resources for which the operation is mentioned will be updated with the specified security tag
+PATCH https://{{hostname}}/$bulk-update
+content-type: application/fhir+json
+Authorization: Bearer {{bearer.response.body.access_token}}
+Prefer: respond-async
+{
+    "resourceType": "Parameters",
+    "parameter": [
+        {
+            "name": "operation",
+            "part": [
+                {
+                    "name": "type",
+                    "valueCode": "upsert"
+                },
+                {
+                    "name": "path",
+                    "valueString": "Patient.meta"
+                },
+                {
+                    "name": "name",
+                    "valueString": "security"
+                },
+                {
+                    "name": "value",
+                    "valueCoding": {
+                        "system": "http://example.org/security-system",
+                        "code": "SECURITY_TAG_CODE",
+                        "display": "Updated Security Tag Display"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "operation",
+            "part": [
+                {
+                    "name": "type",
+                    "valueCode": "upsert"
+                },
+                {
+                    "name": "path",
+                    "valueString": "Observation.meta"
+                },
+                {
+                    "name": "name",
+                    "valueString": "security"
+                },
+                {
+                    "name": "value",
+                    "valueCoding": {
+                        "system": "http://example.org/security-system",
+                        "code": "SECURITY_TAG_CODE",
+                        "display": "Updated Security Tag Display"
+                    }
+                }
+            ]
+        }
+    ]
+}
+
+### Get the status of the bulk update operation
```

Comment:
Could you move the GET and DELETE to the top of the document so they are easier to find? 
Also, could you add a name to all the bulk update calls and use a reference in these urls instead of a hard coded number. There are examples in bulk delete. This will make it easier to run the samples.